No configuration file found.
No pyproject.toml file found.
stubPath /mnt/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 2 source files
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py:21:6 - error: Import "tfx.components.evaluator" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py:22:6 - error: Import "tfx.components.evaluator" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py:23:6 - error: Import "tfx.dsl.components.base" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py:24:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py:25:6 - error: Import "tfx.orchestration.kubeflow.v2.container" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py:26:6 - error: Import "tfx.orchestration.kubeflow.v2.proto" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py:27:6 - error: Import "tfx.types" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor.py:28:6 - error: Import "tfx.utils" could not be resolved (reportMissingImports)
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:22:6 - error: Import "tfx.components.evaluator" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:23:6 - error: Import "tfx.components.evaluator" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:24:6 - error: Import "tfx.dsl.io" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:25:6 - error: Import "tfx.orchestration.kubeflow.v2.container" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:26:6 - error: Import "tfx.types" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:27:6 - error: Import "tfx.types" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:28:6 - error: Import "tfx.utils" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:56:18 - error: "input_dict" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:56:18 - error: Cannot assign member "input_dict" for type "_ArgsCapture"
    Member "input_dict" is unknown (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:57:18 - error: "output_dict" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:57:18 - error: Cannot assign member "output_dict" for type "_ArgsCapture"
    Member "output_dict" is unknown (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:58:18 - error: "exec_properties" is not a known member of "None" (reportOptionalMemberAccess)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:58:18 - error: Cannot assign member "exec_properties" for type "_ArgsCapture"
    Member "exec_properties" is unknown (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:115:28 - error: Cannot access member "input_dict" for type "_ArgsCapture"
    Member "input_dict" is unknown (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:117:28 - error: Cannot access member "output_dict" for type "_ArgsCapture"
    Member "output_dict" is unknown (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/80b0d4447470f6fe637897dece66dc53c0b8cbe7/a/tfx/orchestration/kubeflow/v2/container/kubeflow_v2_run_executor_test.py:119:37 - error: Cannot access member "exec_properties" for type "_ArgsCapture"
    Member "exec_properties" is unknown (reportGeneralTypeIssues)
24 errors, 0 warnings, 0 infos 
Completed in 1.492sec

time cost(in seconds): 1.7242035865783691
